docs/streams/developer-guide/processor-api.html

```diff
@@ -439,6 +439,9 @@ <h2><a class="toc-backref" href="#id8">Connecting Processors and State Stores</a
                 indicating that the state store cannot be found. If the state store is not associated with the processor
                 in the <code class="docutils literal"><span class="pre">Topology</span></code> code, accessing it in the processor&#8217;s <code class="docutils literal"><span class="pre">init()</span></code> method will also throw an exception at
                 runtime, indicating the state store is not accessible from this processor.</p>
+            <p>Note that there could be multiple <code class="docutils literal"><span class="pre">ProcessorContext</span></code> instances initialize your <code class="docutils literal"><span class="pre">Processor</span></code> during initialization.
```

Comment:
I think the first sentence might be too detailed? Maybe we can simplify:
```
<p>Note that the `process()` function takes a `ProcessorSupplier` as argument, and that the supplier pattern requires that a new `Processor` instance is return each time `get()` is called. Creating a single `Processor` object and returning the same object reference in `get()` would be violation of the supplier pattern and leads to runtime exceptions.</p>
```

Reply:
Adopted and revised a little. Thanks!